cupyx/scipy/sparse/linalg/_solve.py

```diff
@@ -78,6 +80,300 @@ def lsqr(A, b):
     return ret
 
 
+def lsmr(A, b, damp=0.0, atol=1e-6, btol=1e-6, conlim=1e8, maxiter=None):
```

Comment:
How about letting it accept `x0` for the initial guess?

Reply:
Yeah, I will add it.